innopoints/schemas/notification.py

```diff
@@ -21,7 +21,7 @@
 
 class PayloadSchema(ma.Schema):
     project = ma.Nested('ProjectSchema', only=('id', 'name', 'review_status', 'lifetime_stage'))
-    activity = ma.Nested('ActivitySchema', only=('id', 'name'))
+    activity = ma.Nested('ActivitySchema', only=('id', 'name', 'internal'))
```

Comment:
Why would the notification care about that? We have a special NotificationType for being added as moderator anyways, and it doesn't even include the activity. No notification that has the activity in its payload goes to anyone but the volunteer.

Reply:
On another thought, though, the `claim_innopoints` notification can be modified to be sent to moderators as well, but will be more like "You got innopoints" as they don't have to submit feedback about their work. If so, we should probably write an issue to implement this.
Otherwise, not needed.

Reply:
`claim_innopoints` will be sent to moderators. And it is important that the frontend knows not to link to an activity that is not there. So this attribute is to determine just that.

Reply:
Actually, the notification will be sent to the moderators as well since it is sent to all applications on all activities.

Reply:
And all of that works out of the box, no further code needed, except for the frontend